test-utils/src/lib.rs

```diff
@@ -180,6 +184,25 @@ pub async fn set_entity_revision_field<'a>(
     Ok(())
 }
 
+pub fn from_value_to_taxonomy_type(value: Value) -> TaxonomyType {
+    let type_camel_case = value.as_str().unwrap();
+    let type_kebab_case = type_camel_case.to_case(Case::Kebab);
+    TaxonomyType::from_str(type_kebab_case.as_str()).unwrap()
+}
+
+pub const TAXONOMY_TYPES_WITHOUT_ROOT: [TaxonomyType; 10] = [
+    TaxonomyType::Subject,
+    TaxonomyType::Topic,
+    TaxonomyType::TopicFolder,
+    TaxonomyType::Blog,
+    TaxonomyType::Curriculum,
+    TaxonomyType::CurriculumTopic,
+    TaxonomyType::CurriculumTopicFolder,
+    TaxonomyType::Forum,
+    TaxonomyType::ForumCategory,
+    TaxonomyType::Locale,
+];
```

Comment:
```suggestion
pub const TAXONOMY_TYPES_WITHOUT_ROOT: [TaxonomyType; 2] = [
    TaxonomyType::Topic,
    TaxonomyType::TopicFolder,
];
```

The others I do not want to support in the future...

Reply:
Good. Changed, also the var name.
BTW: this is in test-utils/lib. But I suppose you would like to put it in production code isn't?